core/cap-0034.md

```diff
@@ -448,7 +483,7 @@ consensus protocol.
 Considering the `closeTime`-related changes in this CAP also raised a further
```

Comment:
Not sure I'm following this sentence, raised a further discussion?

Reply:
I seem to have lost a word there.  Yes, "discussion" might have been it.  :)